go/acct/radius.go

```diff
@@ -41,9 +42,10 @@ func (h *PfAcct) hasher() hash.Hash64 {
 }
 
 func (h *PfAcct) HandleAccountingRequest(w radius.ResponseWriter, r *radius.Request) {
+	ctx := r.Context()
+	defer statsd.NewStatsDTiming(ctx).Send("pfacct.HandleAccountingRequest")
```

Comment:
If you change the name of the timings here, you need to adjust the dashboard so that it uses the new names

Reply:
We still need the old graph since the radius-acct can still be used.